app/models/file_node.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 class FileNode < Valkyrie::Model
+  include Valkyrie::Model::AccessControls
```

Comment:
So it's two things:

1. I'm really torn on having "default attributes" as part of `Valkyrie::Model`. It's the same reason that `ID` isn't inherited. There's something nice about knowing exactly what you've got when you look at a model.

2. When I tried adding these to `Valkyrie::Model` dry::struct got really mad at me.